compiler/rustc_target/src/spec/targets/riscv64gc_unknown_fuchsia.rs

```diff
@@ -1,6 +1,14 @@
-use crate::spec::{CodeModel, SanitizerSet, Target, TargetOptions, base};
+use crate::spec::{CodeModel, SanitizerSet, Target, base};
 
 pub(crate) fn target() -> Target {
+    let mut base = base::fuchsia::opts();
+    base.code_model = Some(CodeModel::Medium);
+    base.cpu = "generic-rv64".into();
+    base.features = "+m,+a,+f,+d,+c".into();
+    base.llvm_abiname = "lp64d".into();
+    base.max_atomic_width = Some(64);
```

Comment:
Should we also set `stack_probes` just like we do for the other targets?

Reply:
done